server/database/schema.sql

```diff
@@ -476,72 +476,83 @@ create table lab12_exercise
-        'https://all.rit.edu/Lab0/', null,
+        'https://all.rit.edu/Lab0/', NULL,
         'In this lab, you will learn how to develop a lab for ALL from beginning to end. Starting with writing a lab, you will explore concepts like ideation, gathering sources, and finally designing the lab’s structure. Afterward, you will be able to implement a lab on both the frontend and backend while upholding best practices and standards. Click “Next” to start!',
         e'{
-	"description":{
-		"header":"",
-		"content":""
-	},
-	"body":[
-		{
-			"header":"Why Do We Build Labs?",
-			"type":"",
-			"content":["Lab 0 Reading Content about the importance of accessibility in software eduction."]
-		},
-		{
-			"header":"",
-			"type":"links",
-			"content":[]
-		},
-		{
-			"header":"What Goes into Building a Lab?",
-			"type":"",
-			"content":["Lab 0 reading on building labs"]
-		},
-		{
-			"header":"",
-			"type":"links",
-			"content":[]
-		},
-		{
-			"header":"Our Tech Stack",
-			"type":"",
-			"content":["Information about tech stack and other important development info"]
-		},
-		{
-			"header":"",
-			"type":"image",
-			"content":{
-				"image":"",
-				"alt":"",
-				"sub_caption":"",
-				"caption":""
-			}
-		},
-		{
-			"header":"",
-			"type":"",
-			"content":[]
-		},
-        {
-			"header":"",
-			"type":"links",
-			"content":["Maybe embed links to documentation for each section of the tech stack, i.e. React, Express, Sequelize, etc"]
-		},
-		{
-			"header":"",
-			"type":"links",
-			"content":[]
-		}
-	],
-	"footer":{
-		"links":[]
-	}
-}', e'  [{"title":"Audio Cues","link":"https://www.youtube.com/embed/vU_Di8EtF3M"},{"title":"Audio Cues Lecture","link":"https://www.youtube.com/embed/Wlf8A0w66o0"}]
+        "description": {
+            "header": "",
+            "content": ""
+        },
+        "body": [
+            {
+                "header": "History of Accessible Learning Labs",
+                "type": "",
+                "content": [
+                    "Accessible Learning Labs, in conjunction with Rochester Institute of Technology and the National Science Foundation, was created in 2019 by principal investors Daniel Krutz and Samuel Malachowsky with the purpose of spreading awareness of the importance of accessible software on the web, as well as the crucial role that software developers play in designing and creating a more accessible, usable internet that anyone can use. Comprised of a fully undergraduate development team, Accessible Learning Labs plays a crucial part in not only creating a better future for the web, but creating more conscientious software engineers that know the importance of accessibility. Since Lab 1 - Accessibility to Sound and Speech released in 2019, Accessible Learning Labs has released 11 more labs, with topics ranging from Accessibility Cognitive Disabilities to AI in Cybersecurity, with more in the works with each passing semester."
```

Comment:
Accessibility to Cognitive Disabilities not Accessibility Cognitive Disabilities